pipeline/inputs/tail.md

```diff
@@ -119,3 +120,43 @@ File rotation is properly handled, including logrotate's _copytruncate_ mode.
 
 Note that the `Path` patterns **cannot** match the rotated files. Otherwise, the rotated file would be read again and lead to duplicate records.
 
+
+## Encoding
+
+Tail input plugin supports also common 8-bit character encodings: 
+
+```
+  windows-1250, windows-1251, windows-1252,
+  windows-1253, windows-1254, windows-1255,
+  windows-1256, windows-1257, windows-1258,
+  
+  iso-8859-1, iso-8859-2, iso-8859-3, iso-8859-4,
+  iso-8859-5, iso-8859-6, iso-8859-7, iso-8859-8,
+  iso-8859-9, iso-8859-10, iso-8859-11, iso-8859-13,
+  iso-8859-14, iso-8859-15, iso-8859-16
+```
+
+By default encoding fails if there are invalid characters in input .  With //... postfix character behaviour can be changed. 
+
+| Encoding | Description  |
+| :--- | :--- |
+| _ENCODING_ | fail if bad characters |
+| _ENCODING_//IGNORE | skip bad characters |
```

Comment:
Ah, still going with the `//MODE` iconv approach, but why?  I think simply specifying the (possibly empty or NULL) replacement string for bad characters suffices for this, no need for iconv baggage.